src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(number) {
+      calculator.result += number;
+    },
+    subtract(number) {
+      calculator.result -= number;
+    },
+    multiply(number) {
+      calculator.result *= number;
+    },
+    divide(number) {
+      calculator.result /= number;
+    },
+    reset(number) {
```

Comment:
The `reset` method currently accepts a `number` parameter, but it is not used. Since the method always resets the result to 0, consider removing the parameter to avoid confusion.